### ISSUE TITLE
[AccessibleValueHandler] Support iOS description with double tap and hold interaction

### ISSUE THREAD

**Issue by zepumph** [phetsims/sun#668]
From https://github.com/phetsims/ratio-and-proportion/issues/232, it was recognized that double-tap and hold way of interacting with a PDOM slider on iOS doesn't support description from AccessibleValueHandler.js. This is a known issue (tagging @jessegreenberg), but the Ratio and Proportion team felt like this was problematic coming from an interview where the user only used double-tap and hold to interact with the ratio hand sliders.

Noting that other mobile iOS implemented sims (OL, GFL(B)) have this same problem.

We don't want to work on this right now, as we don't have the bandwidth this quarter to take this on while we work on getting RaP ready for publication. Tagging @emily-phet @terracoda so they are aware of this issue.

**Comment by zepumph** [comment]
During a brief conversation with @jessegreenberg, he mentioned that we could solve this by changing the focus logic of phet sims. Instead of having a touch event blur the PDOM element, we could keep focus on the PDOM input slider, and then likely aria-valuetext would continue to work for free. Likely context responses wouldn't come along for free, since they are tied to an interaction, instead of the valueProperty changing.

**Comment by zepumph** [comment]
but since `AccessibleValueHandler.alertContextResponse()`, we can just call that to opt into context responses on some DragListener.end call.

**Comment by terracoda** [comment]
This sounds very promising. 
Let's have a look at this again once we have all the issues for this sim sorted out.

**Comment by terracoda** [comment]
Oh, we are in sun, this affects more than just RaP, of course.
It affects all sims with sliders and mobile accessibility.
Lets work on this as we can in between top priorities.

**Comment by emily-phet** [comment]
Hey folks - if I'm understanding [this comment](https://github.com/phetsims/sun/issues/668#issuecomment-724943115) correctly, a proposed solution would allow interaction to occur with the double-tap + hold interaction, providing some description but not all. 

Since we can tell if someone is on a mobile device, perhaps the solution here is a combination of supporting whatever access to the description we can, plus added text associated with the sliders, etc. that indicating the preferred way of interacting (that will provide the full description). 

If the changes that would enable some description to be provided for double-tap + hold does not fit into this quarter, let's just add some description indicating the swiping method for interaction (that provides full description), and investigation/work to be planned for in our 2021 planning meetings.

**Comment by terracoda** [comment]
@emily-phet, my understanding of https://github.com/phetsims/sun/issues/668#issuecomment-724944562 is that it might be straight forward to get both object and context responses using the Accessible Value handler to call context responses.

**Comment by jessegreenberg** [comment]
In https://github.com/phetsims/scenery/issues/1094 a change was made so that most events with iOS VoiceOver go through pointer events rather than click events. So this issue is going to be important to get the general flow with iOS VoiceOver working correctly.

It also has overlap with other upcoming features like getting focus highlights to persist while using a mouse or finger. I think this should be worked on now.

**Comment by jessegreenberg** [comment]
I am working on this change in https://github.com/phetsims/scenery/issues/1137, when testing a slider I am finding that iOS 13 and 14 VoiceOver does not read the new value or the aria-valuetext when operating a slider with the double-tap and drag operation. I tested with this example outside of a PhET sim:
<details>

```html
<!DOCTYPE html>
<html>
<head>
  <meta charset="UTF-8">
  <title>title</title>
</head>
<body>

<div>
  <input type="range" id="slider">
  <p id="readout"></p>
</div>


</body>

<script>
  const sliderElement = document.getElementById( 'slider' );
  sliderElement.addEventListener( 'change', () => {
    sliderElement.setAttribute( 'aria-valuetext', 'New Value: ' + slider.value );
    document.getElementById( 'readout' ).innerText = 'Value: ' + sliderElement.value;
  } );
</script>
</html>
```
</details>

I tried both `change` and `input` events in this example, neither worked. Furthermore, I wasn't able to move the slider with double-tap and hold gesture after the first move of the thumb because iOS VoiceOver isn't sending the pointer events to the right place (the location of the thumb).

If we want this case to be supported we will need to submit the aria-valuetext as alerts. But this feels like a failure of the screen reader, and I don't think we should do work to support this.

Assigning to @terracoda for further discussion.

**Comment by terracoda** [comment]
I wonder if we could provide a plat-form specific alert to use swipes to use the object. At some point in the future, iOS VoiceOver should be able to deliver aria-valuetext naturally like macOS VoiceOver.

**Comment by terracoda** [comment]
And can we submit a bug report to Apple?

**Comment by jessegreenberg** [comment]
Discussed with TS and MK:

- We will not go any farther with a custom solution here since this is the behavior with iOS VoiceOver and sliders at the moment.
- I will submit a bug report to apple.
- If we need to convey this to users it is the kind of thing we could put in teacher tips.
